packages/python-generator-core/README.md

```diff
@@ -0,0 +1,17 @@
+# Core Python Generator
```

Comment:
So is this just a python library that gets called from `cadl-python` or `@autorest/python`? if so should it really be a npm package, why not be a python package?